twingate/client_resource.go

```diff
@@ -21,6 +21,46 @@ type Resource struct {
 	IsActive        graphql.Boolean
 }
 
+type gqlResource struct {
+	IDName
+	Address struct {
+		Type  graphql.String
+		Value graphql.String
+	}
+	RemoteNetwork struct {
+		ID graphql.ID
+	}
+	Groups struct {
+		PageInfo struct {
+			HasNextPage graphql.Boolean
+		}
+		Edges []*Edges
+	} `graphql:"groups(first: $first)"`
```

Comment:
`$first` is not a good name as its very generic - what if we had multiple different connections we wanted to paginate here?
This should be something like `$groupsFirst` or `$groupsPageSize`

Reply:
good point, thank you

Reply:
updated